modules/apps/object/object-test/src/testFunctional/paths/ObjectCustomViews.path

```diff
@@ -275,6 +275,16 @@
 		<td>//div[contains(@id,'clay-dropdown-menu')]//input[contains(@class,'toggle-switch-check')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>AGGREGATION_FILTER_VALUE</td>
+		<td>//input[contains (@type, 'number')]</td>
+		<td></td>
+	</tr>
+	<tr>
+		<td>PICKLIST_FILTER_KEBAB_MENU_OPTION</td>
```

Comment:
Is there no generic path for this? If not, can we change the name? Only because Objects team also handles the Picklist feature so it could be a little confusing.

Reply:
I couldn't find a generic path for this specific filter menu option, I'll edit the path name for this so it won't have the picklist in it.